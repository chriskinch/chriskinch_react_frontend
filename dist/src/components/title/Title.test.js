'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _Title = require('./Title');

var _Title2 = _interopRequireDefault(_Title);

var _reactTestRenderer = require('react-test-renderer');

var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

it('renders correctly', function () {
  var tree = _reactTestRenderer2.default.create(_react2.default.createElement(_Title2.default, { text: 'My example title' })).toJSON();
  expect(tree).toMatchSnapshot();
});

//# sourceMappingURL=Title.test.js.map