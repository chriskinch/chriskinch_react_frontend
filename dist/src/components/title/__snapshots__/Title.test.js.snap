// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="block block-core block-page-title-block"
  id="block-chriskinch-page-title"
>
  <h1
    className="page-title"
  >
    My example title
  </h1>
</div>
`;
